test/hotspot/jtreg/gc/g1/TestGCLogMessages.java

```diff
@@ -265,7 +265,7 @@ private void testConcurrentRefinementLogs() throws Exception {
     LogMessageWithLevel exhFailureMessages[] = new LogMessageWithLevel[] {
         new LogMessageWithLevel("Recalculate Used Memory", Level.DEBUG),
         new LogMessageWithLevel("Restore Preserved Marks", Level.DEBUG),
-        new LogMessageWithLevel("Remove Self Forwards", Level.DEBUG),
+        new LogMessageWithLevel("Restore Retained Regions", Level.DEBUG),
```

Comment:
Please also check the work item in this test. Needs to be added just here.

Reply:
Thanks Thomas, I have added checking newly added work items.